artemis-hawtio/artemis-console/pom.xml

```diff
@@ -55,18 +55,6 @@
       <version>${hawtio.version}</version>
       <scope>provided</scope>
     </dependency>
-    <dependency>
-      <groupId>com.google.guava</groupId>
-      <artifactId>guava</artifactId>
-      <!-- License: Apache 2.0 -->
-      <!-- see https://github.com/google/guava/issues/3006: checker-compat-qual could be excluded -->
-      <exclusions>
-        <exclusion>
-          <groupId>org.checkerframework</groupId>
-          <artifactId>checker-compat-qual</artifactId>
-        </exclusion>
-      </exclusions>
-    </dependency>
```

Comment:
This was here to override the version of guava in the repackaged hawtio war for the console. See the exclusion further down that is still present. Think we will need to check what happens to the management console here, its possible this may have broken it, but at least removed the version override.

Reply:
Content of artemis-console-2.30.0-SNAPSHOT.war\WEB-INF\lib :
checker-qual-3.5.0.jar
commons-codec-1.15.jar
commons-io-2.11.0.jar
commons-logging-1.2.jar
error_prone_annotations-2.3.4.jar
failureaccess-1.0.1.jar
hawtio-core-2.15.0.jar
hawtio-system-2.15.0.jar
hawtio-util-2.15.0.jar
httpclient-4.5.13.jar
httpcore-4.4.13.jar
j2objc-annotations-1.3.jar
jolokia-core-1.7.0.jar
json-20171018.jar
json-simple-1.1.1.jar
jsr305-3.0.2.jar
listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar

Reply:
Right...Guava isnt there, but is is meant to be there. Note its dependencies still being present.

The exclusion I mentioned earlier, in the repackaging the module build does, removed the version the original war included. You have removed the dependency that lead to its replacement in the repackaged war.

Reply:
`checker-compat-qual` is not there
BTW artemis-console-2.29.0.war\WEB-INF\lib\ contains
checker-qual-3.12.0.jar
checker-qual-3.5.0.jar

Should be `checker-qual` excluded then?

Reply:
The checker-compat-qual is presumably not there because it was excluded, per the comment in your removal. Having 2 checker-qual versions would seem like an oversight though, not noticed during an update.

Reply:
same for now - no `checker-compat-qual` in war

Reply:
Right, the original war doesnt have it (but does have guava), so it was also excluded from the repackaged war to match. The original war has checker-qual-3.5.0.jar, which should presumably have been excluded when unpacking it, due to the newer guava bringing in checker-qual-3.12.0.jar